source/Calamari.Common/Features/Packages/ExtractPackage.cs

```diff
@@ -23,16 +23,16 @@ public ExtractPackage(ICombinedPackageExtractor combinedPackageExtractor, ICalam
             this.log = log;
         }
 
-        public void ExtractToStagingDirectory(PathToPackage? pathToPackage, IPackageExtractor? customPackageExtractor = null)
+        public void ExtractToStagingDirectory(PathToPackage? pathToPackage, IPackageExtractor? customPackageExtractor = null, string? workingDirectory = null)
         {
-            var targetPath = Path.Combine(Environment.CurrentDirectory, "staging");
+            var targetPath = Path.Combine(workingDirectory ?? Environment.CurrentDirectory, "staging");
```

Comment:
Whats the reason that this step needs to extract the package to a working directory rather than the Current Directory?  Based on the implimentation of the value being passed in up the chain it looks like it will typically resolve to Environment.CurrentDirectory anyway?

Reply:
Ah yeah this was for tests, but is now not required - I might back out these workingDirectory changes.

Reply:
This has been reverted now.